cime/config/e3sm/tests.py

```diff
@@ -329,4 +329,19 @@
         "time"    : "01:00:00",
     },
 
+    "e3sm_scream" : {
+        "time"  : "03:00:00",
+        "tests" : (
+            "SMS_D.ne4_ne4.FSCREAM-HR",
+            "SMS_D.ne4_ne4.FSCREAM-LR",
+            "ERS.ne4_ne4.FSCREAM-HR",
+            "ERS.ne4_ne4.FSCREAM-LR",
+            "ERP.ne4_ne4.FSCREAM-HR",
```

Comment:
Do we need both ERS and ERP tests? Also, have you run the FSCREAM-HR test with ne4? I believe it will crash.

Reply:
@brhillman  - Are you saying that if ERP (exact restart w/ thread change) fails, ERS would have failed as well so we might as well only do ERP? 

And in terms of FSCREAM-HR crashing, are you thinking that lack of parameterized deep convection will leave ne4 unable to transport heat upward, causing huge pulses of upward motion? I thought the model generally ran stably without deep convection even though the climate looked horrible... but that's at ne30. If possible, I'd really like to include HR tests if possible so we can quickly catch bugs specific to HR... I'm not set up to submit these HR tests, but it would be good if someone tried them before we merge this PR.